lib/trace-writer.js

```diff
@@ -120,14 +120,10 @@ TraceWriter.prototype.flushBuffer_ = function(projectId) {
   // Privatize and clear the buffer.
   var buffer = this.buffer_;
   this.buffer_ = [];
+  var joined = buffer.join();
+  buffer = '[' + joined.slice(0, joined.length -1) + ']';
```

Comment:
Good catch, this was a carry over from the fully string based buffering. It's been eliminated.